src/components/DetailSidebar/styles.ts

```diff
@@ -21,10 +18,14 @@ export const TitleListItemText = styled(ListItemText)({
   },
 });
 
-export const TitleAvatar = styled(Avatar)({
+export const PackageDescription = styled('span')({
+  '&&': {
```

Comment:
`<div>` is block by default, you might switch to `<div>` to avoid this CSS. Somehow, I guess this is related with my comment above. So, l wait for the context.

Reply:
It is !
See my comment above :)